consensus/bor/bor.go

```diff
@@ -430,15 +439,6 @@ func (c *Bor) verifyCascadingFields(chain consensus.ChainReader, header *types.H
 		for i, validator := range currentValidators {
```

Comment:
We should use this `if-block` for `(number-1)  % c.config.Sprint == 0`. 

This checks if the incoming block header has valid validator bytes or not.

Reply:
Done.